common/lib/xmodule/xmodule/library_tools.py

```diff
@@ -96,6 +96,17 @@ def _filter_child(self, usage_key, capa_type):
         assert isinstance(descriptor, CapaDescriptor)
         return capa_type in descriptor.problem_types
 
+    def can_use_library_content(self, course_id):
+        """
+        Determines whether a modulestore holding a course_id supports libraries.
+        """
+        store = self.store
+        # Skip MixedModuleStore abstraction.
+        if hasattr(store, '_get_modulestore_for_courseid'):
+            # pylint: disable=protected-access
+            store = store._get_modulestore_for_courselike(course_id)
+        return hasattr(store, "copy_from_template")
```

Comment:
n.b. `_get_library` is a private method and is not part of the service's interface.